presto-native-tests/src/test/java/com/facebook/presto/nativetests/AbstractTestAggregationsNative.java

```diff
@@ -38,15 +38,15 @@ public void init(String storageFormat, boolean sidecarEnabled)
             charTypeUnsupportedError = ".*Unknown type: char.*";
             timeTypeUnsupportedError = ".*Unknown type: time.*";
             approxDistinctUnsupportedSignatureError = ".*Unexpected parameters \\(timestamp with time zone.*\\) for function.*";
-            mergeAggFunctionUnsupportedError = ".*Unexpected parameters \\(qdigest.* for function native.default.merge.*";
+            mergeAggFunctionUnsupportedError = ".*Scalar function native.default.value_at_quantile not registered with arguments.*";
```

Comment:
nit: Could we rename `mergeAggFunctionUnsupportedError` to `valueAtQuantileFunctionUnsupportedError2` ?

Interesting that the error message for `value_at_quantile(merge(qdigest_agg(CAST(orderkey AS DOUBLE))), 0.5E0)` in testcases `testStatisticalDigestMerge` and `testStatisticalDigest` does not match and we can check this separately.